<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnewuser</class>
 <widget class="QDialog" name="addnewuser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>591</width>
     <height>397</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add User</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string>Student ID:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="idTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="passwordTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="accountNameLabel">
      <property name="text">
       <string>Account Name: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="fNameLabel">
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lNameLabel">
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="cityLabel">
      <property name="text">
       <string>City: </string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="zipLabel">
      <property name="text">
       <string>Zip Code:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Email Address:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="phoneLabel">
      <property name="text">
       <string>Phone Number:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="accountNameTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="fNameTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lNameTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="addressTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="cityTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="zipTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="emailTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="phoneTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addnewuser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addnewuser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
